Python/_warnings.c

```diff
@@ -132,6 +133,8 @@ _PyWarnings_InitState(PyInterpreterState *interp)
 {
     WarningsState *st = &interp->warnings;
 
+    st->mutex = (struct _PyMutex){ 0 };
+
```

Comment:
This isn't needed: `WarningsState` (and the rest of `PyInterpreterState`) is zero initialized. For most of `PyInterpreterState` we don't explicitly initialized the fields to zero.

Up to you if you want to keep it.

Reply:
Good point; I removed it.